src/api/graphql.ts

```diff
@@ -1,9 +1,14 @@
-const graphqlEndpoint = `${window.origin}/graphql`;
+const graphqlEndpoint = `https://qual-it.ecom.wilson.com/en-us/graphql`;
+const basicToken = 'd2lsc29udGVhbXNob3AtZGV2OndpbHMwbnRlYW1zaDBwLUQzVg==';
```

Comment:
Use api key in dev_template instead ?
Api key : 95689dc242b0433da022462d27547b30

Reply:
Yes, this should come in the StoreDetailsProps, which is for local dev filled with data from dev_template and when integrated in the main repo is coming from the configs.json